project-management/src/main/java/life/qbic/projectmanagement/domain/model/project/Contact.java

```diff
@@ -42,6 +50,13 @@ public String emailAddress() {
     return emailAddress;
   }
 
+  public String oidc() {
```

Comment:
Is this value always present? What is the empty value? A little javadoc would be appreciated.